### DISCUSSION TITLE
Default Selected / Checked Radio Button?

### DISCUSSION THREAD

**Discussion by ChiragSukhala** [orchidsoftware/platform#1588] [Q&A]
'Advanced form controls' features  RadioButtons but neither that page nor document specifies how to select or check a default value.

**Comment by S-senj** [comment]
I also did not find a description of this field in the documentation. After digging into the source code, I found the `checked()` method and passed the class property there, which I defined in the `query()` method.
```php
...
Radio::make('field_name')->checked($this->isChecked)
...
```
The method takes a boolean value.

**Reply by yetki** [reply]
didnt work at all on V14

**Comment by c-gross** [comment]
I'm using value to set a default:
```php
CheckBox::make('whatever.status')
                ->value(1)
                ->title('Status')
                ->sendTrueOrFalse()
```
